server/src/startup/main.cpp

```diff
@@ -61,6 +61,7 @@
 #include "unittest/gpos/memory/IMemoryPoolTest.h"
 #include "unittest/gpos/memory/CMemoryPoolBasicTest.h"
 #include "unittest/gpos/memory/CCacheTest.h"
+#include "unittest/gpos/memory/CMemoryPoolCustomAllocTest.h"
```

Comment:
Please order the `#include` if possible to help speed up compilation time. Thanks.